lib/src/screen_lock.dart

```diff
@@ -172,20 +185,40 @@ class _ScreenLockState extends State<ScreenLock> {
     retries++;
   }
 
-  Widget buildHeadingText() {
-    if (widget.confirmation) {
-      return StreamBuilder<bool>(
-        stream: inputController.confirmed,
-        builder: (context, snapshot) {
-          if (snapshot.hasData && snapshot.data!) {
-            return widget.confirmTitle;
-          }
-          return widget.title;
-        },
+  Widget makeDelayBuilder(Duration duration) {
+    if (widget.delayBuilder != null) {
+      return widget.delayBuilder!(duration);
+    } else {
+      return HeadingTitle(
+        text:
+            'Input locked for ${(duration.inMilliseconds / 1000).ceil()} seconds.',
       );
     }
+  }
+
+  Widget buildHeadingText() {
+    return StreamBuilder<Duration>(
```

Comment:
I thought about it.

We could only use this streambuilder if inputDelay is not `Duration.zero`